aldryn_newsblog/views.py

```diff
@@ -21,7 +21,7 @@ class ArticleDetail(TranslatableSlugMixin, AppConfigMixin, DetailView):
     slug_field = 'slug'
 
     def get_queryset(self):
-        return Article.objects.active_translations(
+        return Article.published.active_translations(
             translation.get_language()
         ).filter(
             app_config__namespace=self.namespace
```

Comment:
AppHookConfigTranslatableManager is relatively new feature. Let's try to use it in a separate pull request.